.github/workflows/issues-chores.yml

```diff
@@ -51,7 +51,29 @@ jobs:
           github-token: ${{ secrets.GH_ISSUES_PRS_CHORES }}
           labeled: Area:QA
 
-      - name: Attach all tickets with label Area:QA to the Security Roadmap Project
+      - name: Attach all tickets with label Area:Security and incident to the Security Roadmap Project
+        uses: actions/add-to-project@v1.0.2
+        id: new-security-incident
+        with:
+          project-url: https://github.com/orgs/Jahia/projects/22
+          github-token: ${{ secrets.GH_JAHIACI_ISSUES_CHORES }}
+          labeled: Area:Security, incident
+          label-operator: AND
+
+      - name: Create a Security issue in the Support Jira
```

Comment:
Do you have a mechanism to make sure you're not creating the jira ticket for every single events on the issue workflows ?

Reply:
Yes, but this workflow is potentially called for every single event happening on the issue. What would happen if the issue indeed has these labels, but someone decided to add an additional label at a later stage.